share/tools/web_config/webconfig.py

```diff
@@ -1315,17 +1314,19 @@ def do_POST(self):
             return self.send_error(403)
         self.path = p
 
-        ctype, pdict = cgi.parse_header(self.headers["content-type"])
+        # This is cheesy, we want just the actual content-type.
+        # In some cases it'll give us the encoding as well,
+        # ("application/json;charset=utf-8")
+        # but we don't currently care.
+        ctype = self.headers["content-type"].split(";")[0]
 
-        if ctype == "multipart/form-data":
-            postvars = cgi.parse_multipart(self.rfile, pdict)
```

Comment:
Probably just make this `return self.send_error(500)`, just in case.

Reply:
Yeah, that's a good idea.

Reply:
Done in 17501bcc5750574c66722cde2560ae9643b97e3a.